dd-sdk-android-gradle-plugin/src/main/kotlin/com/datadog/gradle/plugin/internal/sanitizer/SanitizerStrategy.kt

```diff
@@ -0,0 +1,5 @@
+package com.datadog.gradle.plugin.internal.sanitizer
+
+internal interface SanitizerStrategy {
```

Comment:
Why do you need this specific interface. It has the same purpose and signature as `UrlSanitizer`.

Reply:
I'd like to keep things separated in case we want to go back to this later. This feature might need improvements later or new strategies to handle different other cases so it will be easily scalable this way. It is also better for testing as I can handle the `Strategy` output better.

Reply:
Nevermind...I see what you're saying...will remove that and make this as a wrapper of `UrlSantizers` implementations.